pywinauto/unittests/test_application.py

```diff
@@ -1204,6 +1204,30 @@ def test_find_elements_re(self):
         self.assertTrue(len(windows) >= 1)
 
 
+if UIA_support:
+    class UIAWindowSpecificationTestCases(unittest.TestCase):
+        """Unit tests for the application.Application class with UIA backend"""
+
+        def setUp(self):
+            """Set some data and ensure the application is in the state we want"""
+            Timings.defaults()
+            self.app = Application(backend="uia").start("Notepad")
```

Comment:
In this test file we use `_notepad_exe()` to get an exe with correct bitness. For "uia" it doesn't matter so much. But I'd prefer unified style for all tests in this script.